fix: `Element.attachShadow()`'s options is missing an attribute. (#20197)
mdn/content: files/en-us/web/api/element/attachshadow/index.md

      - : A boolean that, when set to `true`, specifies behavior that mitigates custom element issues around focusability.
        When a non-focusable part of the shadow DOM is clicked, the first focusable part is given focus, and the shadow host is given any available `:focus` styling.

    - `slotAssignment`

>       - : default: `named`.
>       - : An enum type, it has two attribute values, `manual` and `named`. When set `slotAssignment` to `manual`, we need to use {{DOMxRef("HTMLSlotElement.assign()")}} to manual assign `slot`.

Reviewer: ```suggestion
      - : Either `manual` or `named` (default). When set to `manual`, use {{DOMxRef("HTMLSlotElement.assign()")}} to assign a value to`slot`.
```

Reviewer: Thank you for the submission. This is more typical of how we do this.
